build/configs/imxrt1020-evk/tc_protected/defconfig

```diff
@@ -663,7 +663,6 @@ CONFIG_SMARTFS_MAXNAMLEN=32
 CONFIG_SMARTFS_ALIGNED_ACCESS=y
 # CONFIG_SMARTFS_DYNAMIC_HEADER is not set
 CONFIG_SMARTFS_JOURNALING=y
```

Comment:
since you removed the code for journaling with your PR, and in defconfigs you have it enabled.  What should be the behavior? I think if you are creating a separate PR for removing JOURNALING, and to add a new mechanism will take months, better do a clean removal or dont do it at all.

Reply:
We gonna implement new journaling, so this conifg will be used again.
Make sure this is not master branch, will be merged when we finished.